python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -92,8 +93,12 @@ def __init__(self, port, public_ip, with_credentials=False):
             if self.nbvm is None and not with_credentials:
                 self.cors = CORS(app)
                 self.with_credentials = False
-                self.ip = LOCALHOST
-                self.env = 'local'
+                if public_ip is not None:
+                    self.ip = public_ip
+                    self.env = PUBLIC_VM
+                else:
+                    self.ip = LOCALHOST
+                    self.env = 'local'
```

Comment:
Might be out of PR scope, but since you already have a constant for `'public_vm'` will you also create one for `'local'`?

Reply:
I never re-use the local constant, I think it is just there for debugging